### PR TITLE
Papp 422 | Fix v8 upgrade issues

### PR DESCRIPTION
**Pull request by pankaj443**
v7 -> v8 Migration
- getOrientation move to android/ios driver fn
- MobileElement class is removed use RemoteWebElemet instead
- upgrading to v8.6.0 as older version had some bugs and 8.6.0 is latest in v8 series

### CODE REVIEW

File: src/main/java/io/percy/appium/metadata/AndroidMetadata.java [hunk 1]
```diff
@@ -96,4 +96,28 @@ public String getDisplaySysDump() {
...
+                  return driver.getOrientation().toString().toLowerCase();
+              } catch (java.lang.NoSuchMethodError e) {
+                  return "portrait";
+              }
+          } else {
+              return "portrait";
+          }
+      } else {
+          Object orientationCapability = driver.getCapabilities().getCapability("orientation");
+          if (orientationCapability != null) {
```
Comment (prklm10) [inline src/main/java/io/percy/appium/metadata/AndroidMetadata.java:115]: Why have we moved this function from metadata to here? It seems repetative.

Comment (prklm10) [reply src/main/java/io/percy/appium/metadata/AndroidMetadata.java:115]: also take master merge

Comment (pankaj443) [reply src/main/java/io/percy/appium/metadata/AndroidMetadata.java:115]: in v8 getOrientation is not on parent lvl. its moved to respective classes.

Comment (prklm10) [reply src/main/java/io/percy/appium/metadata/AndroidMetadata.java:111]: Ack, Can you add a comment. above

**Review by prklm10 — APPROVED** [review]
LGTM

**Review by rishigupta1599 — APPROVED** [review]

**Pull request merged**